File: python/1113.py
while True:
    try:
        vls=input().split(' ')
        x=int(vls[0])
        y=int(vls[1])
        if x>y:
            print('Decrescente')
        elif x<y:
            print('Crescente')
    except EOFError:
        break

File: javascript/1113.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
while(lines.length){
    [x,y]=lines.shift().trim().split(' ').map((el) => parseInt(el))
    if(x===y)break
    x>y?console.log("Decrescente"):console.log("Crescente");
}